### PR TITLE
Update Subnautica (change Wine version, use DXVK)

### PR DESCRIPTION
**Pull request by Zemogiter**
Another performance update to this script. Tested and ready for review.

### CODE REVIEW

File: Applications/Games/Subnautica/resources/fix.reg [hunk 1]
```diff
@@ -3,8 +3,8 @@ REGEDIT4
...
-[HKEY_CURRENT_USER\Software\Wine\AppDefaults\Steam.exe]
-"Version"="winxp64"
- 
-[HKEY_CURRENT_USER\Software\Wine\AppDefaults\steamwebhelper.exe]
-"Version"="winxp64"
+[HKEY_CURRENT_USER\Software\Wine]
+"HideWineExports"="Y"
+
+[HKEY_CURRENT_USER\Software\Wine\Direct3D]
+"csmt"="0x00000000"
```
Comment (plata) [inline Applications/Games/Subnautica/resources/fix.reg]: Use the csmt plugin instead.

File: Applications/Games/Subnautica/resources/fix.reg [hunk 2]
```diff
@@ -3,8 +3,8 @@ REGEDIT4
 [HKEY_CURRENT_USER\Software\Wine\X11 Driver]
 "UseTakeFocus"="N"
 
-[HKEY_CURRENT_USER\Software\Wine\AppDefaults\Steam.exe]
-"Version"="winxp64"
- 
-[HKEY_CURRENT_USER\Software\Wine\AppDefaults\steamwebhelper.exe]
-"Version"="winxp64"
+[HKEY_CURRENT_USER\Software\Wine]
+"HideWineExports"="Y"
```
Comment (plata) [inline Applications/Games/Subnautica/resources/fix.reg]: What does this do? Should we have a plugin for it?

File: Applications/Games/Subnautica/resources/fix.reg (hunk 1 above)
Comment (Zemogiter) [reply Applications/Games/Subnautica/resources/fix.reg]: CSMT was used to increase performance in some applications. Its now depreciated.

File: Applications/Games/Subnautica/resources/fix.reg (hunk 2 above)
Comment (Zemogiter) [reply Applications/Games/Subnautica/resources/fix.reg]: It hides wine version from appliactions in wineprefix. It was needed at some point before DXVK became a viable option. Now I've tried to run Subnautica without it and it made no difference. Very few games need wine-staging and of those games maybe one or two really needs this setting to be enabled. We can always make a plugin for that but it's far from being a top priority.

Comment (plata) [reply Applications/Games/Subnautica/resources/fix.reg]: I see. But if the script doesn't need it, we should just remove it.

**Pull request merged**